public/app/plugins/datasource/graphite/func_editor.js

```diff
@@ -68,22 +69,42 @@ function (angular, _, $) {
             }
           }
 
+          function paramDef(index) {
+            if (index < func.def.params.length) {
+              return func.def.params[index];
+            }
+            if (_.last(func.def.params).multiple) {
+              return _.assign({}, _.last(func.def.params), {optional: true});
+            }
+            return {};
+          }
+
           function inputBlur(paramIndex) {
             /*jshint validthis:true */
             var $input = $(this);
+            if ($input.data('typeahead') && $input.data('typeahead').shown) {
+              return;
```

Comment:
what was the reason for this condition? It causes most blur scenarios to not work any more